src/components/ConnectPin.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Connect Pin component that renders a basic profile
+ * image of the user to be used as MapView.Marker custom view.
+ *
+ * @prop pinType        - either "veteran" or "parterOrg"
+ */
+
+import React from 'react';
+import { StyleSheet, View, Image } from 'react-native';
+
+export default class ConnectPin extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+    };
+  }
+
+  renderPin() {
+    return this.props.pinType == 'parterOrg' ? (
+      <Image
+        source={require('../../assets/images/pin-green.png')}
+        style={styles.pin}
+      />
+    ) : (
+      <Image
+        source={require('../../assets/images/pin.png')}
+        style={styles.pin}
+      />
```

Comment:
are we differentiating the PO vs veteran pins here, by color (green vs white)?

Reply:
can we also make it clear where the user him/herself is located in relation to all pins?

Reply:
yup and yup I will add a current location indicator